test/commands/test/generate/uitest-test.ts

```diff
@@ -155,26 +141,19 @@ describe("Validating UITest template generation", () => {
       });
     });
 
-    const packageFilePath = path.join(command.outputPath, `AppCenter.UITest.${command.platform}/packages.config`);
     const projectFilePath = path.join(
       command.outputPath,
       `AppCenter.UITest.${command.platform}/AppCenter.UITest.${command.platform}.csproj`
     );
 
     // Assert
-    let packageFileContent = await pfs.readFile(packageFilePath, "utf8");
-    expect(packageFileContent).not.contain(fakeLatestVersion);
-
     let projectFileContent = await pfs.readFile(projectFilePath, "utf8");
     expect(projectFileContent).not.contain(fakeLatestVersion);
 
     // Act
     await (command as any).processTemplate();
 
     // Assert
-    packageFileContent = await pfs.readFile(packageFilePath, "utf8");
```

Comment:
why we had so many dups?

Reply:
It is tests. We read file, then execute command, then read file again to make sure the changes were applied.

Reply:
We have dups within this PR, since initially we did not remove the instances associated with `packagePath`, but replaced them with `projectPath`.

Reply:
why to remove it now?

Reply:
We don't have this path after migrating `csproj` file to SDK style.

Reply:
But same path is used on line 178 no?

Reply:
On line 178 we use projectFilePath(csproj file) and on the remote packageFilePath(packages.config file)